src/manage/modify-attribute-table/index.md

```diff
@@ -0,0 +1,18 @@
+# How to Modify Attribute Table
```

Comment:
How to Modify Data Schema (Attribute Table)?

I am bit lost, is this article about avoiding conflicts? or just how to update the schema? looks like the first, but title is unrelated to text? Also you are referencing the file that I do not see?